warehouse/query-core/src/main/java/datawave/query/jexl/functions/ContentFunctionsDescriptor.java

```diff
@@ -424,9 +424,12 @@ public static JexlNode distributeFunctionIntoIndexQuery(JexlNode function, JexlN
             List<JexlNode> components = new LinkedList<>();
             
             JexlNode deref = JexlASTHelper.dereference(indexQuery);
-            if (deref instanceof ASTAndNode) {
-                // single component
+            if (deref instanceof ASTAndNode || deref instanceof ASTEQNode) {
+                
+                // it is possible that an index query contains only a single equality node
+                // i.e., "phrase('foo', 'foo')" would produce a single equality node "FOO == 'bar'"
```

Comment:
Is this comment correct?  `phrase(foo, foo)` becomes `FOO == 'bar'` ?

Reply:
I'll make this language more precise. The index expansion for that function will produce a single equality node.

Reply:
my question was more about it going from foo, foo, to foo, bar.  is that right?